__tests__/app.test.js

```diff
@@ -75,6 +75,44 @@ describe("GET/api/articles", () => {
         expect(body.results).toBeSortedBy("created_at", { descending: true });
       });
   });
+  describe("GET /api/articles(queries => topic, sort_by order)", () => {
+    test("200: returns the correct articles when provided with a topic query", () => {
+      return request(app)
+        .get("/api/articles?topic=cats")
+        .expect(200)
```

Comment:
need to do a 200 to check a path with orderby
another 200 when correct topic but no articles so it gives back an empty array
a 400 for invalid sortby
a 400 for invalid orderby